IN-CORE/incore-playbook
# More configuration #48

This is ready to be reviewed. @Rashmil-1999 @navarroc @Vismayak @ywkim312 

To test:
- Follow the [README](https://github.com/IN-CORE/incore-playbook#readme) session "Example of use this repository in child repositories"
- Child repository try SLC playbook https://github.com/IN-CORE/SLC-Playbook; Check out SLC playbook branch https://github.com/IN-CORE/SLC-Playbook/pull/1
- Varying the configuration in the child repo to see changes. e.g.
```
store.dispatch(setAllowedHazardTypes(["earthquake", "tornado", "hurricane"]));
```

--------------
include configurables of:
- datawolf
- globals
- assets
- allowedHazards and allowedDataType

To test:
- use SLC repository to test: https://github.com/IN-CORE/SLC-Playbook
